webpack/ForemanTasks/Components/TasksDashboard/Components/StoppedTasksCard/StoppedTasksCard.js

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card } from 'patternfly-react';
+import classNames from 'classnames';
+import { translate as __ } from 'foremanReact/common/I18n';
+import './StoppedTasksCard.scss';
+
+class StoppedTasksCard extends React.Component {
```

Comment:
This component should be a `functional` component since it dosesn't use state or lifeCycle methods.

Reply:
added a helpers file for the components functions